keyboards/ferris/sweep/config.h

```diff
@@ -70,6 +57,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define SOFT_SERIAL_PIN D2
 
 #define EE_HANDS
+#define SPLIT_USB_DETECT
```

Comment:
Are you sure you want this as a default for AVR boards? It's default behaviour for ARM stuff anyway.

Reply:
I think that'll help with occasional quality issues with cheap AVR controllers. Are there any downside for MCUs that don't need it (other than 4-byte space cost)?

Reply:
It can cause an issue when booting or resuming from sleep with some computers. If it doesn't establish a USB connection within `SPLIT_USB_TIMEOUT` both sides will end up as a slave meaning you'll need to replug or reboot the keyboard.

Reply:
Thanks for the inputs—that directive is removed as default.